data/sql/updates/pending_db_world/rev_1585996243671367200.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1585996243671367200');
+
+-- fix Julianne's first death sound
+UPDATE `creature_text` SET `Sound`=9198 WHERE `CreatureID`=17534 and `GroupID`=2 and `ID`=0;
```

Comment:
when you write SQL commands use uppercase also for "AND"

Reply:
@aleigood i fixed the "and" for you, next time no :D